stdlib/asm/sys/mod.masm

```diff
@@ -33,3 +33,16 @@ end
 export.drop_stack_top
     dropw dropw dropw dropw
 end
+
+#! Logs a precompile commitment and removes the helper words produced by `log_precompile`.
+#!
+#! Input: `[COMM, TAG, ...]`
+#! Output: Stack without the three helper words `[R1, R0, CAP_NEXT]`.
+#! Cycles: 1 (plus cost of the underlying `log_precompile` instruction).
+export.log_precompile_request
```

Comment:
The way to write inputs/outputs here would be:
```
Inputs:  [COMM, TAG]
Outputs: []
```
Also, the cycle count is not correct. By my count this procedure takes 19 cycles (including the 1 cycle for the `log_precompile` instruction).